app/templates/Tags/index.php

```diff
@@ -0,0 +1,48 @@
+<?php
+/**
+ * @var \App\View\AppView $this
+ * @var iterable<\App\Model\Entity\Tag> $tags
+ */
+?>
+<div class="tags index content">
+    <?= $this->Html->link(__('New Tag'), ['action' => 'add'], ['class' => 'button float-right']) ?>
+    <h3><?= __('Tags') ?></h3>
+    <div class="table-responsive">
+        <table>
+            <thead>
+                <tr>
+                    <th><?= $this->Paginator->sort('id') ?></th>
+                    <th><?= $this->Paginator->sort('title') ?></th>
+                    <th><?= $this->Paginator->sort('created') ?></th>
+                    <th><?= $this->Paginator->sort('modified') ?></th>
+                    <th class="actions"><?= __('Actions') ?></th>
+                </tr>
+            </thead>
+            <tbody>
+                <?php foreach ($tags as $tag): ?>
+                <tr>
+                    <td><?= $this->Number->format($tag->id) ?></td>
+                    <td><?= h($tag->title) ?></td>
+                    <td><?= h($tag->created) ?></td>
+                    <td><?= h($tag->modified) ?></td>
+                    <td class="actions">
+                        <?= $this->Html->link(__('View'), ['action' => 'view', $tag->id]) ?>
+                        <?= $this->Html->link(__('Edit'), ['action' => 'edit', $tag->id]) ?>
+                        <?= $this->Form->postLink(__('Delete'), ['action' => 'delete', $tag->id], ['confirm' => __('Are you sure you want to delete # {0}?', $tag->id)]) ?>
+                    </td>
+                </tr>
+                <?php endforeach; ?>
+            </tbody>
+        </table>
+    </div>
+    <div class="paginator">
+        <ul class="pagination">
+            <?= $this->Paginator->first('<< ' . __('first')) ?>
+            <?= $this->Paginator->prev('< ' . __('previous')) ?>
+            <?= $this->Paginator->numbers() ?>
+            <?= $this->Paginator->next(__('next') . ' >') ?>
+            <?= $this->Paginator->last(__('last') . ' >>') ?>
+        </ul>
+        <p><?= $this->Paginator->counter(__('Page {{page}} of {{pages}}, showing {{current}} record(s) out of {{count}} total')) ?></p>
+    </div>
```

Comment:
This section repetitive with https://github.com/combimauri/tutorial-cakephp/blob/690421ebeb1995bfa109ae2baafd26fff6ccd235/app/templates/Users/index.php#L38-L47

Please try to create an element for the paginator and make it reusable between 2 template files. 

Please refer https://book.cakephp.org/4/en/views.html#elements on how to make a reusable element.